### PR TITLE
Feat cart

### PR DESCRIPTION
**Pull request by krishanu2911**
Added the cart page for the maple coffee
Also Added a few util classes which will be moved to CL in future.

https://user-images.githubusercontent.com/75560307/154391442-89f61543-1a6e-4919-b11a-4812e5aa8ee1.mp4

### CODE REVIEW

File: pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://maple-ui.netlify.app/CSS/base.css">
```
Comment (tanishq20) [inline pages/cart.html]: You can import this style link directly in `style.css`. Like this, you will not have to give link in every HTML file.

**Review by tanishq20 — COMMENTED** [review]
Well-written clean code. Suggested minor change.

File: pages/cart.html (hunk 1 above)
Comment (krishanu2911) [reply pages/cart.html]: Thank Tanishq for suggesting this updated the required changes

File: CSS/cart.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.cart-display{
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (nishasen) [inline CSS/cart.css]: use utils for this

File: CSS/cart.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    gap: 2rem;
+    padding-top: 4rem;
+    margin-bottom: 4rem;
+}
+.cart-card-footer{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```
Comment (nishasen) [inline CSS/cart.css]: use utils class for this, it will reduce your code

File: pages/cart.html [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+        </section>
+        <!-- ******************* -->
+        <footer class="ecom-footer maple-flex-xy-center">
+            <div>
+                Made with love ❤ By <a class="semibold-font footer-link" target="_blank" href="https://twitter.com/KrishanuDutta_">KrishanuDutta</a>
+            </div>
+        </footer>
+    </main>
+    <script src="https://kit.fontawesome.com/0e624d796c.js" crossorigin="anonymous"></script>
+    <script src="https://kit.fontawesome.com/7e9e9f492e.js" crossorigin="anonymous"></script>
```
Comment (nishasen) [inline pages/cart.html:116]: Any reason why you gave two fontawesome links

**Review by nishasen — COMMENTED** [review]
The code looks clean also I have suggested some changes please look into the comments.

File: pages/cart.html (hunk 4 above)
Comment (krishanu2911) [reply pages/cart.html:116]: I was having some problem with fontawesome like some icons were not loading with the previous link so have created the newlink with fontawesome will sort out this problem in the coming days for now using both links.

File: CSS/cart.css (hunk 2 above)
Comment (krishanu2911) [reply CSS/cart.css]: Thank you Nisha for reminding I have already declared in util but forget to use that.

File: CSS/cart.css (hunk 3 above)
Comment (krishanu2911) [reply CSS/cart.css]: Done

**Review by tanishq20 — APPROVED** [review]

File: pages/cart.html (hunk 4 above)
Comment (nishasen) [reply pages/cart.html:116]: Okay

**Comment by nishasen** [comment]
Now it looks nice, good work

**Review by nishasen — APPROVED** [review]

**Pull request merged**